_inc/client/state/site/reducer.js

```diff
@@ -149,6 +178,15 @@ export function getAvailablePlans( state ) {
 	return get( state.jetpack.siteData, [ 'data', 'sitePlans' ] );
 }
 
+/**
+ * Returns all wpcom products
+ * @param  {Object}  state Global state tree
+ * @return {Object}  Products
+ */
+export function getAllProducts( state ) {
```

Comment:
Another question here: should this filter out any products that we're not interested in?

Reply:
I think that could be done on the api side as well.

Reply:
Yes I think this is an enhancement we should make on the API side. The Jetpack plugin can filter the products before sending them back to the server.

Reply:
Makes sense, but can we rename this to just `getProducts` then so it's closer to what it does?

```suggestion
export function getProducts( state ) {
```